documentation/howto/import_FH_app.adoc

```diff
@@ -0,0 +1,189 @@
+= Import Your FeedHenry Application into the IDE
+:page-layout: howto
+:page-tab: docs    
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+Use Red Hat JBoss Developer Studio’s `Import` wizard to import a FeedHenry application into your workspace. When imported, edit the application as required and push it back to the FeedHenry server. 
+
+Follow the provided steps to import and make changes to a FeedHenry application using your Red Hat JBoss Developer Studio workspace:
+
+. <<import_app, Import Your FeedHenry Application>>
+. <<change_app, Change the Application>>
+. <<test_app, Test the Application>>
+. <<push_changes_to_FH, Push the Changes Back to the FeedHenry Server>>
+
+== Prerequisites 
+Ensure that the following prerequisites are met to enable the FeedHenry feature in JBoss Developer Studio:
+
+* <<enable_FH, Enable the FeedHenry Feature>>
+* <<set_preferences, Set the Preferences for your application import>>
+
+[[enable_FH]]
+=== Enable the FeedHenry Feature 
+To enable the FeedHenry feature:
+
+. In JBoss Central, click the `Software/Update` tab.
+. In the Features Available` list, select the `Hybrid Mobile Tools + CordovaSim` check box and then click `Install/Update`.
+
+.Hybrid Mobile Tools + CordovaSim Check Box Selected
+image::hybrid_mobile_tools__cordovasim_check_box_selected_1.png[Hybrid Mobile Tools + CordovaSim Check Box Selected]
+
+. Follow the on-screen instructions to complete the installation.
+
+[[set_preferences]]
+=== Set the Preferences for your application import
+To set the preferences:
+
+. Click `Window` > `Preferences`.
+. In the `Preferences` window, in the search field type, `FeedHenry` and press Enter.
+. Complete the following fields in the `Preferences` window:
+** Ensure that the `Target URL` field displays the URL to your FeedHenry server. The URL to your FeedHenry server should be something like: `https://_[your-studio-domain]_.feedhenry.com`. 
+** In the `API Key` field, copy and paste the API key from the FeedHenry website.
+
+.Setting Preferences for FeedHenry
+image::preferences.png[Setting Preferences for FeedHenry]
+
+*Note:* The user can either use an existing API key or generate a new one to set the preferences. To obtain the API key from the FeedHenry website click `Settings` > `API Key Management`. To generate a new API key, click `Settings` > `API Key Management`, and then click `Add New Key`. 
+
+[start=4]
+. Click `Apply` and then click `OK` to close the `Preferences` window.
+
+
+[[import_app]]
+== 1. Import Your FeedHenry Application
+
+Ensure that your JBoss Developer Studio preferences are set before importing a FeedHenry application. If not set yet, JBoss Developer Studio will prompt you to set the preferences. These preferences are set once when importing an application for the first time and the configured preferences are used for all imports in the future. 
```

Comment:
It is not "JBoss Developer Studio" preferences, it is the very specific preferences that one need to set.